docs/publish-strategy.md

```diff
@@ -1,10 +1,14 @@
-# HuggingFace Publish Strategy for InstructLab
+# Publishing Strategy for InstructLab
 
-This document describes the publishing strategy used for all models in the [InstructLab](https://huggingface.co/instructlab) organization.
+This document describes the publishing strategy used for all models in the InstructLab HuggingFace [organization](https://huggingface.co/instructlab) and the InstructLab Ollama [user account](https://ollama.com/instructlab).
 
 ## What are we publishing and why?
 
-The InstructLab team will be periodically training the full unquantized model with new Pull Requests to the [taxonomy](https://github.com/instructlab/taxonomy) repository. When the evaluation shows that the model has improved, the team will be publishing an unquantized model and a 4-bit quantized GGUF form model to a platform called HuggingFace.
+The InstructLab team will be periodically training the full unquantized model with new Pull Requests to the [taxonomy](https://github.com/instructlab/taxonomy) repository. When the evaluation shows that the model has improved, the team will be publishing 4-bit quantized GGUF form model to a platform called Ollama and both an unquantized model and a 4-bit quantized GGUF form model to a platform called HuggingFace.
+
+## What is Ollama?
+
+[Ollama](https://ollama.com/) is a platform that allows users to run open-source large language models locally on their machines. Ollama covers a broad spectrum of users, from seasoned AI professionals to people looking to explore the potential of AI and makes it easier for users to leverage the power of LLMs without having to rely on a cloud infrastructure. Since Ollama only supports publishing GGUF models, we will only be publishing the 4-bit quantized versions of our merlinite and granite models to the InstructLab Ollama user account.
```

Comment:
```suggestion
[Ollama](https://ollama.com/) is a platform that allows users to run open-source large language models locally on their machines. Ollama covers a broad spectrum of users, from seasoned AI professionals to people looking to explore the potential of AI and makes it easier for users to leverage the power of LLMs without having to rely on a cloud infrastructure. Since Ollama only supports publishing GGUF models, we will only be publishing the 4-bit quantized versions of our Merlinite and Granite models to the InstructLab Ollama user account.
```

Reply:
@nathan-weinberg https://github.com/instructlab/dev-docs/pull/72

Reply:
@alinaryan if you fix this one capitalization, I think this is good to go!

Reply:
@cdoern @nathan-weinberg Addressed!